suite-native/ethereum-tokens/src/ethereumTokensSelectors.ts

```diff
@@ -0,0 +1,40 @@
+import { A, G } from '@mobily/ts-belt';
+
+import { AccountsRootState, selectAccountByKey } from '@suite-common/wallet-core';
+import { TokenInfo } from '@trezor/blockchain-link';
+
+import { EthereumTokenAccountWithBalance } from './types';
+
+export const filterTokenHasBalance = (token: TokenInfo) => !!token.balance && token.balance !== '0';
```

Comment:
no need to export this

Reply:
[0a1bb8e](https://github.com/trezor/trezor-suite/pull/7757/commits/0a1bb8e03a8d45650350886eca8a1f4fabdf455a)